src/GHC.hs

```diff
@@ -34,10 +34,10 @@ defaultKnownPackages =
     [ array, base, binary, bytestring, cabal, checkApiAnnotations, compareSizes
     , compiler, containers, deepseq, deriveConstants, directory, dllSplit
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal
-    , ghcCompact, ghci, ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs
+    , ghcCompact, ghci, ghcPkg, ghcPrim, ghcTags, haddock, hsc2hs
     , hp2ps, hpc, hpcBin, integerGmp, integerSimple, iservBin, libffi
     , mkUserGuidePart, mtl, parsec, parallel, pretty, primitive, process, rts
-    , runGhc, stm, templateHaskell, terminfo, text, time, touchy, transformers
+    , runGhc, stm, templateHaskell, text, time, touchy, transformers
```

Comment:
What happened with `haskeline` and `terminfo`?

Reply:
`terminfo` can't be compiled because lack of dependencies on the target platform IIRC. And `haskeline` depends on it. In GHC's tree, you can find `WITH_TERMINFO=NO` in `mk/flavours/quick-cross.mk`, which disables it.

For `haskeline`, it is more or less a quick hack:

```
libraries/haskeline_CONFIGURE_OPTS += --flags=-terminfo
```

I should fix `haskeline` like above rather than turning it completely off. Thanks!

Reply:
Thanks for the clarification!